packs/chef/actions/lib/shellhelpers.py

```diff
@@ -43,17 +42,18 @@ def __init__(self, parser_options):
             self.parser.add_argument(k, lg, **kwargs)
             self._keyname[lg.lstrip('-')] = k
 
+
     def parse(self, argv=None):
-        argv = argv or sys.argv
```

Comment:
Also it's a bit ironic. That devs of python stdlib who were also truly inspired with "explicit better than implicit paradigm" made such intricate behavior for parse_args )))